addons/point_of_sale/tools/posbox/overwrite_after_init/etc/nginx/sites-enabled/default

```diff
@@ -76,16 +76,16 @@ server {
 # You can move that to a different file under sites-available/ and symlink that
 # to sites-enabled/ to enable it.
 #
-#server {
-#       listen 80;
+server {
+       listen 80;
```

Comment:
is the 8069 still available? should be consider redirecting to 80?

Reply:
8069 is still available, but it points to Odoo and it seemed easier to make the image available through nginx.